libs/openFrameworks/types/ofColor.cpp

```diff
@@ -346,7 +346,7 @@ float ofColor_<PixelType>::getBrightness() const {
 
 template<typename PixelType>
 float ofColor_<PixelType>::getLightness() const {
-	return (r + g + b) / 3.f;
```

Comment:
@ofTheo 
if `PixelType` is `float` and rgb-values are too large (near the `numeric_limit<float>::max()`, but is not `inf`), then `(r + g + b) / 3.f` will be `inf` and `r / 3.f + g / 3.f + b / 3.f` will be not `inf`.
maybe `PixelType` is `unsigned char` or `unsigned short` then `(r + g + b) / 3.f` and `r / 3.f + g / 3.f + b / 3.f` are almost same. 
because type of `(r + g + b)` will glow to `int` implicitly, maybe `int` is enough to calculate sum of 3 values of `PixelType`.

(i don't know too large value will set into r, g, b...)